apps/dapp-console-api/src/Service.ts

```diff
@@ -103,6 +105,8 @@ export class Service {
    */
   public static readonly init = async () => {
     const redisClient = new Redis(envVars.REDIS_URL)
+    // replace redis URL with gateway URL
+    const redisCache = new RedisCache(envVars.REDIS_URL)
```

Comment:
nit: if this is specifically meant for reading from gateway we should consider naming this variable `gatewayRedisCache` to avoid confusion